Makefile

```diff
@@ -76,9 +76,12 @@ dist: build_linux build_win
 	mkdir -p dist
 	mkdir -p dist/$(DISTDIR)-windows-amd64/bin-windows-amd64
 	cp README.md dist/$(DISTDIR)-windows-amd64
-	cp bin-windows-amd64/csaf_uploader.exe bin-windows-amd64/csaf_checker.exe bin-windows-amd64/csaf_downloader.exe dist/$(DISTDIR)-windows-amd64/bin-windows-amd64/
+	cp bin-windows-amd64/csaf_uploader.exe bin-windows-amd64/csaf_validator.exe \
+bin-windows-amd64/csaf_checker.exe bin-windows-amd64/csaf_downloader.exe \
```

Comment:
Indent this, please.

Reply:
Done in https://github.com/csaf-poc/csaf_distribution/pull/327/commits/89f463f982c3a456b7a4c4faffd8f31ba0e6b302